src/Model/ModelManagerInterface.php

```diff
@@ -156,40 +156,38 @@ public function getUrlsafeIdentifier($model);
      *
      * @param string $class
      *
-     * @return mixed
+     * @return object
      */
     public function getModelInstance($class);
 
     /**
      * @param string $class
      *
-     * @return array|\ArrayAccess
+     * @return array
```

Comment:
If this method is really capable to return an instance of `\ArrayAccess`, maybe we should use `iterable` instead.

Reply:
So we should remove all `array` annotations and replace them with iterable?

Reply:
After testing what I've proposed, I think these docblocks must be kept as they are, since `ArrayAccess` is also not interpreted as `iterable` (https://3v4l.org/oXbXV).

Reply:
Reverted